src/entities/MovieForm/MovieForm.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "react";
+import "./MovieForm.css";
+import { genresList } from "../../entities/SelectTabs/selectTabsMock";
+
+export const MovieForm = ({ initialMovie, onFormSubmit }) => {
+  const [movie, setMovie] = useState({
+    title: initialMovie?.title ?? "",
+    movieUrl: initialMovie?.movieUrl ?? "",
+    genre: initialMovie?.genre ?? "",
+    overview: initialMovie?.overview ?? "",
+    releaseDate: initialMovie?.releaseDate ?? "",
+    rating: initialMovie?.rating ?? "",
+    runtime: initialMovie?.runtime ?? "",
+    overview: initialMovie?.overview ?? "",
+  });
+
+  const handleFieldChange = (event) => {
+    setMovie((prevMovie) => ({
+      ...prevMovie,
+      [event.target.name]: event.target.value,
+    }));
+  };
+
+  return (
+    <form className="movie-form" action="#">
```

Comment:
I believe we should have `onSubmit` here

Reply:
done